packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -757,10 +757,8 @@ class MarkdownBuilder implements md.NodeVisitor {
       case 'pre':
         return styleSheet.codeblockAlign;
       case 'hr':
-        print('Markdown did not handle hr for alignment');
```

Comment:
Can we use `debugPrint` instead of just deleting?

Reply:
Sure thing. I thought of so, and I agree with the decision. I believe @goderbauer specified deleting but the rationale behind it was to solve analysis warnings. 

I'll change these to debug prints.